src/Blablacar/Redis/Client.php

```diff
@@ -54,7 +54,11 @@ public function connect()
         $this->redis = new \Redis();
         $this->redis->connect($this->host, $this->port);
         if (null !== $this->base) {
-            $this->redis->select($this->base);
+            try {
+                $this->redis->select($this->base);
+            } catch (\RedisException $e) {
+                throw new \RedisException($e->getMessage() . " " . $this->host, $e->getCode(), $e);
```

Comment:
I'd also put the port and add braces something like:
`sprintf('%s (%s:%d)', $e->getMessage(), $this->host, $this->port);`